### ISSUE TITLE
v1.100 - Version string and Unique font identifier fields errors

### ISSUE THREAD

**Issue by kenmcd** [d-sargent/platypi#2]
There seems to be an issue with some variables.
The actual version number is missing.
Below is what is actually in the Regular font file.

Unique font identifier (NameID 3): `%d.%03d;BOLT;Platypi-Regular`

Version string (NameID 5): `Version %d.%03d; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.33]`

Same thing in the other styles.

**Comment by d-sargent** [comment]
Thanks again for being my tech checker @kenmcd I appreciate it! This is my first typeface so I am making it up as I go, putting out fires and causing others :)

I put in a custom version string because I was getting a Fontbakery error with the default version number.

Can I ask how you are viewing this info? I'm getting some other Fontbakery errors and my current process is to try different settings, output fonts, then rerun the test. But if I knew how to see this info I might get a better grasp on what is going on.

**Comment by kenmcd** [comment]
> Can I ask how you are viewing this info?

I first noticed this using Windows Font Viewer.
When viewing TTF fonts it displays the Version String under the font name.
So then I checked the name table in OT Master Light v3.7.0 (free).
https://web.archive.org/web/20200612160827/https://www.fontmaster.nl/legacy/downloads_rdrct.html
It is good for checking things because it only displays exactly what is in there - it changes nothing (unlike font editors).
You can also use `fonttools` `ttx` feature to dump the font to XML, and then look at the `name` table info (as-is unchanged).

I assumed you put some sort of date variables or something in the Glyphs application.
I am on Windows so not real familiar with how Glyphs works.
But I figure that is where you will need to fix this (in Glyphs).

-----
I did notice a couple other things...

The `ffi` ligature has no Unicode code point assigned.
Should be `FB03`

Some of the characters which should be composites are not.
For example check these `ğǧĝġ`
/gbreve/gcaron/gcircumflex/gdotaccent
And the `caron` in `gcaron` should be `caroncomb`
You should check all the characters with diacritics as this is something GF will check when they review.
Take a look at the GF Diacritics docs (and the links at the bottom).

**Comment by d-sargent** [comment]
@kenmcd I owe you a coffee/beer!

The `ffi` ligature is an easy fix, thanks for spotting that.

I use an alternate `g` with a modified ear for `ğǧĝġ`. Maybe I've set these up incorrectly in Glyphs. More troubleshooting for me to do there.

The `gcaron` is a head-scratcher. In Glyphs it uses the `caroncomb`... so maybe it is exporting incorrectly? I'll do some experiments.

I tried the OT Master Light v3.7.0 software, but unfortunately, it doesn't run on a modern-day Mac. I'll see if I can find an equivalent.  

Thanks again!
